pandas/io/json/json.py

```diff
@@ -108,6 +115,19 @@ def _format_axes(self):
             raise ValueError("Series index must be unique for orient="
                              "'{orient}'".format(orient=self.orient))
 
+    def write(self):
```

Comment:
if you refactor the super-class to make ``.write()`` be ``._write(self, .......)`` (IOW to accept parameters), then these 2 can just be a super call